tonic/models/vic/grid_params.py

```diff
@@ -447,43 +627,68 @@ def make_grid(grid_file, soil_file, snow_file, veg_file, vegl_file,
     the parameter data, if nc_file = False, the dictionary of grids is
     returned.
     """
-    print('making grided parameters now...')
+    print('making gridded parameters now...')
 
-    soil_dict = soil(soil_file)
+    soil_dict = soil(soil_file, c=Cols(nlayers=nlayers,
+                     organic_fract=organic_fract,
+                     spatial_frost=spatial_frost,
+                     spatial_snow=spatial_snow,
+                     july_tavg_supplied=july_tavg_supplied))
+
+    if not cells:
+        cells = len(soil_dict['gridcell'])
 
     if snow_file:
-        snow_dict = snow(snow_file, soil_dict)
+        snow_dict = snow(snow_file, soil_dict, c=Cols(snow_bands=snow_bands))
     else:
         snow_dict = False
 
+    if vegl_file:
+        veglib_dict, lib_bare_idx = veg_class(vegl_file,
+                                    c=Cols(veglib_fcan=veglib_fcan))
+        veg_classes = len(veglib_dict['Veg_class'])
+    else:
+        veglib_dict = False
+        lib_bare_idx = -1
```

Comment:
I'd rather see `lib_bare_idx = None` (or `False`) when you are trying to indicate that this value hasn't been provided.